autorest/namer/name_converter.py

```diff
@@ -71,13 +71,39 @@ def _convert_extensions(operation: Dict[str, Any]) -> None:
                 "x-python-custom-pager-async", "azure.core.async_paging.AsyncItemPaged"
             )
         if operation_extensions.get("x-ms-long-running-operation"):
+            # poller
             operation["extensions"]["poller-sync"] = operation_extensions.get(
                 "x-python-custom-poller-sync", "azure.core.polling.LROPoller"
             )
             operation["extensions"]["poller-async"] = operation_extensions.get(
                 "x-python-custom-poller-async", "azure.core.polling.AsyncLROPoller"
             )
 
+            # polling methods
+            sync_polling_method_directive = "x-python-custom-default-polling-method-sync"
+            operation["extensions"]["default-polling-method-sync"] = {
+                "azure-arm": operation_extensions.get(
+                    sync_polling_method_directive, "azure.mgmt.core.polling.arm_polling.ARMPolling"
+                ),
+                "data-plane": operation_extensions.get(
+                    sync_polling_method_directive, "azure.core.polling.base_polling.LROBasePolling"
+                ),
+            }
+            async_polling_method_directive = "x-python-custom-default-polling-method-async"
+            operation["extensions"]["default-polling-method-async"] = {
+                "azure-arm": operation_extensions.get(
+                    async_polling_method_directive, "azure.mgmt.core.polling.async_arm_polling.AsyncARMPolling"
+                ),
+                "data-plane": operation_extensions.get(
+                    async_polling_method_directive, "azure.core.polling.async_base_polling.AsyncLROBasePolling"
+                ),
+            }
+
+            operation["extensions"]["default-no-polling-method-sync"] = "azure.core.polling.NoPolling"
```

Comment:
I've added the default no polling, and the base polling method. to the namer as well. I'm not exposing a directive to override these (at least not yet, there seems to be no use case for it). But I wanted all of the LRO paths to be in one area, and it cleaned up the code in the LROOperation model class